src/squishy/core.clj

```diff
@@ -3,6 +3,8 @@
             [cemerick.bandalore :as sqs]
             [clojure.tools.logging :as log]))
 
+(def ^:dynamic *visibility-timeout* 30) ; seconds; the SQS default
```

Comment:
Yeah, it should be a per-queue value since it's heavily affected by the queue's visibility timeout setting. I'll make that change.